File: front_pfe/src/app/model/interviewer.ts

export class Interviewer {
    idInterviewer: number;
    firstName : string;
    lastName : string;
    poste : string;

    constructor( idInterviewer: number, firstName : string,lastName : string,poste : string) {
        this.idInterviewer = idInterviewer;
        this.firstName = firstName;
        this.lastName = lastName;
        this.poste = poste;
    }

}

File: back_pfe/src/main/java/com/app/entity/Interviewer.java
package com.app.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import javax.persistence.*;
import java.util.Date;

@AllArgsConstructor
@Entity
@Table(name= "interviewers")
@Getter
@Setter
@ToString
public class Interviewer {
    @Id
    @GeneratedValue
    private int idInterviewer;
    private String firstName;
    private String lastName;
    private String poste;


    public Interviewer() {

    }
}
